### ISSUE TITLE
re:dash should support comments on queries

### ISSUE THREAD

**Issue by fbertsch** [mozilla/redash#279]
Recently I was reviewing a very hefty query in a Databricks notebook, and was able to go through and highlight specific pieces and leave comments about those parts of the query. I think this would go a long way to making query writing, and data@moz, more reproducible and correct.

Related: https://github.com/getredash/redash/issues/4

**Comment by arikfr** [comment]
I assume you mean comments like you can comment on a Google Docs document? Marking specific areas and commenting on them?

**Comment by fbertsch** [comment]
@arikfr right, with support for responses and checking them off as resolved. I would say the Github reviews are probably a more similar feature set.

**Comment by arikfr** [comment]
👍 

I wonder if it makes sense to integrate with GitHub, to offload the collaboration here and preserve familiar workflows (pull requests).
